src/python/review/inspectors/common.py

```diff
@@ -0,0 +1,23 @@
+from math import floor
+
+
+def convert_percentage_of_value_to_lack_of_value(percentage_of_value: float) -> int:
+    """
+    Converts percentage of value to lack of value.
+    Calculated by the formula: floor(100 - percentage_of_value).
+
+    :param percentage_of_value: value in the range from 0 to 100.
+    :return: lack of value.
+    """
+    return floor(100 - percentage_of_value)
+
+
+# TODO: When upgrading to python 3.9+, replace it with removeprefix.
+# See: https://docs.python.org/3.9/library/stdtypes.html#str.removeprefix
```

Comment:
@nbirillo, you can upgrade to python 3.9 if needed.
https://github.com/hyperskill/hyperstyle/blob/9a6d53cd1ca220d97d296c0087056b5885b26281/Dockerfile.base#L1

Reply:
I think we can do it in the next PR since it is a frozen brunch and we only fix bugs, without new features